build-tools/src/main/kotlin/org/jetbrains/kotlin/CopySamples.kt

```diff
@@ -42,14 +60,22 @@ open class CopySamples: Copy() {
         return this
     }
 
-    companion object {
-        val replacements = listOf(
-            "mavenCentral()" to "maven { url 'https://cache-redirector.jetbrains.com/maven-central' }",
```

Comment:
Minor: There is a syntax working in both `*.gradle` and `*.kts`:

```
 maven { setUrl("...") }
```

It seems possible to use it in the `replacement` list and get rid of `replacementsWithWrapper` and duplicated code in the copy task.

Reply:
Thanks, `setUrl` works nice here. But it's not possible to get rid of the duplicated code in the copy task beacuse in `.gradle` and `.gradle.kts` maven setters are different. You still have to wrap original URL, or change them to `setUrl` in all samples, or rewrite them all to `kts`

Reply:
> You still have to wrap original URL, or change them to setUrl in all samples, or rewrite them all to kts

But we don't need to find and replace the whole `maven { url '<some url>' }` expression. It seems enough to replace only the URL used in such expressions just like it is done in current master. The only special cases are `mavenCentral()` and `jcenter()`. In these cases we can use the `setUrl` method to avoid creating different replacements for `*.gradle` and `*.kts`.

Reply:
Yes, it is enough to replace URL only. My fault. I'll fix this